cmake/Modules/Documentation.cmake

```diff
@@ -26,15 +25,63 @@ if(BUILD_DOC)
     COMMAND ${DOCENV_BINARY_DIR}/pip install --upgrade ${LAMMPS_DOC_DIR}/utils/converters
   )
 
+  # download mathjax distribution and unpack to folder "mathjax"
+  file(DOWNLOAD "https://github.com/mathjax/MathJax/archive/3.0.5.tar.gz"
+    "${CMAKE_CURRENT_BINARY_DIR}/mathjax.tar.gz"
+    EXPECTED_MD5 5d9d3799cce77a1a95eee6be04eb68e7)
+
+  if(NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/mathjax)
+    execute_process(COMMAND ${CMAKE_COMMAND} -E tar xzf mathjax.tar.gz WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+    file(GLOB MATHJAX_VERSION_DIR ${CMAKE_CURRENT_BINARY_DIR}/MathJax-*)
+    execute_process(COMMAND ${CMAKE_COMMAND} -E rename ${MATHJAX_VERSION_DIR} ${CMAKE_CURRENT_BINARY_DIR}/mathjax)
+  endif()
```

Comment:
I would use `FetchContent` would be better here, but might be only supported with >=CMake-3.11.

Reply:
there will have to be more updates to this script when PR #1970 gets ready for merge. Let's discuss the best way of doing this then and there. I will be updating  `doc/Makefile` in the process as well and then what is done there can be mirrored in CMake or a simpler version be applied, e.g. a validated version of the tar file can be deposited on https://download.lammps.org/thirdparty/

Reply:
o, and indeed lit looks like FetchContent is not available in CMake 3.10, but that should not stop us from implementing the cleaner future-proof version and fall back to this hack until we raise the required minimum CMake version in a couple of years or so.